test/TestPublicResolver.js

```diff
@@ -453,6 +453,27 @@ contract('PublicResolver', function (accounts) {
                 null
             );
         });
+
+        it('can set specific contenthash type', async () => {
+            await resolver.setContenthash(node, '0x0100000000000000000000000000000000000000000000000000000000000001', '0x01', {from: accounts[0]});
```

Comment:
@filips123 Apologies for looking over your shoulder, was reading the EIP and saw a note there about the unit tests. In truffle overloaded methods can be accessed via a `methods` key with their ABI sig. 

An example from these tests is calling `resolver.setAddr(node, accounts[1], {from: accounts[0]})` as:

https://github.com/ensdomains/resolvers/blob/8958e4fa2d7bcf0e9a6c0806ddf4a2485e93c896/test/TestPublicResolver.js#L126